scripts/generate_config_yaml.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/python3
+
+import argparse
+import os
+import random
+from string import Template
+
+from ament_index_python import get_package_share_directory
```

Comment:
We need to add `<exec_depend>ament_index_python</exec_depend>` to the `package.xml` for this.

Reply:
https://github.com/ros2/buildfarm_perf_tests/pull/29/commits/9dae6c9197677da497dda081d05ae2f8b533d988